src/csharp/Grpc.Tools/Grpc.Tools.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Grpc.Tools</id>
+    <title>gRPC C# Tools</title>
+    <summary>Tools for C# implementation of gRPC - an RPC library and framework</summary>
+    <description>Precompiled protobuf compiler and gRPC protobuf compiler plugin for generating gRPC client/server C# code. Binaries are available for Windows, Linux and MacOS.</description>
+    <version>$version$</version>
+    <authors>Google Inc.</authors>
+    <owners>grpc-packages</owners>
+    <licenseUrl>https://github.com/grpc/grpc/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/grpc/grpc</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <releaseNotes>Release $version$</releaseNotes>
+    <copyright>Copyright 2015, Google Inc.</copyright>
+    <tags>gRPC RPC Protocol HTTP/2</tags>
+	<dependencies>
+	<dependency id="Grpc" version="$version$" />
+	</dependencies>
+  </metadata>
+  <files>
+    <file src="build\Grpc.Tools.targets" target="build\Grpc.Tools.targets" />
```

Comment:
Here's a thought: 

Instead of trying to add this functionality to Grpc.Tools (and possibly breaking users), we could as a first step add a contrib package (e.g. `Grpc.Tools.MSBuildIntegration` ) that would contain your changes. Than users could opt-in to use the msbuild integration. Later, once that solution proves itself useful, we will be in better position to decide how exactly we'd like to integrate it with the other official Grpc packages.